File: app/src/main/java/com/visiontutor/app/Requirment.java
package com.visiontutor.app;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;

public class Requirment extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_requirment);
        Button b1;
        b1 = findViewById(R.id.reqbutton);

        b1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i = new Intent(getApplicationContext(), RequirmentForm.class);
                startActivity(i);


            }
        });

    }
}

File: app/src/main/java/com/visiontutor/app/Requirment.kt
package com.visiontutor.app

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button

import androidx.appcompat.app.AppCompatActivity

class Requirment : AppCompatActivity() {

    lateinit var b1: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_requirment)
        b1 = findViewById(R.id.reqbutton)

        b1.setOnClickListener {
            val i = Intent(applicationContext, RequirmentForm::class.java)
            startActivity(i)
        }

    }
}
